app/javascript/controllers/companion_window_controller.js

```diff
@@ -7,22 +7,33 @@ export default class extends Controller {
                      "downloadModal", "accessibility", "modalComponentsPopover"]
   connect() {
     this.element.hidden = false
+    const classList = this.leftDrawerTarget.classList
+    // Open the drawer on load
+    this.openLeftDrawer(classList)
     EmbedThis.init();
   }
 
   toggleLeft() {
     const classList = this.leftDrawerTarget.classList
     if (classList.contains('open')) {
-      classList.remove('open')
-      setTimeout(() => {
-        this.leftDrawerTarget.style.visibility = 'hidden' // remove from accessability tree
-      }, 500)
+      this.closeLeftDrawer(classList)
     } else {
-      this.leftDrawerTarget.style.visibility = ''
-      classList.add('open')
+      this.openLeftDrawer(classList)
     }
   }
 
+  openLeftDrawer(classList) {
```

Comment:
It feels a little awkward passing classList as an argument here since we're accessing `this.leftDrawerTarget` elsewhere in the method. Can we make this method parameter-less?